### ISSUE TITLE
Write-DbaDbTableData - Example not working

### ISSUE THREAD

**Issue by andreasjordan** [sqlcollaborative/dbatools#7019]
Can someone please have a quick look at this example:
https://github.com/sqlcollaborative/dbatools/blob/a691a9b1338121e11a8872250649ae5fdc9b5507/functions/Write-DbaDbTableData.ps1#L109-L115

![image](https://user-images.githubusercontent.com/66946165/101375900-37699d00-38b0-11eb-9238-b2744bff9292.png)

Why use ` @{L="owner_sid";E={$_."owner_sid"}}`?
And `$tableName` is set but not used.
And the code is not displayed correctly on the docu-website.

**Comment by andreasjordan** [comment]
Ok, it's the owner_sid, which is a byte-array.

This works:
```
$dataset | Select-Object name, create_date, @{L="owner_sid";E={-join $_."owner_sid".ForEach({$_.ToString('X2')})}} | Write-DbaDbTableData -SqlInstance 'localhost,1417' -SqlCredential $containerCred -Database tempdb -Table myTestData -Schema dbo -AutoCreateTable
```

I don't know: Should we change the example to this? Or should we just leave out the owner_sid?

Feedback is welcome!

**Comment by wsmelton** [comment]
doc issue reference: sqlcollaborative/docs#46

**Comment by wsmelton** [comment]
There is no reason that I could see explicitly selecting the properties. If you just pipe that to the command though it still fails, just with a different error.

![image](https://user-images.githubusercontent.com/11204251/102677407-e0917c80-4167-11eb-8427-4dac0432252b.png)

I do recall most of the examples on this command worked properly so not sure what changed in the command. I would see this as the command needing to handle the `varbinary` type properly from a PowerShell perspective so the user does not have to.

**Comment by andreasjordan** [comment]
This will probably fail because of owner_sid as well. I think all varbinary columns are received as bytes-array and because of the array are not iConvertable. (Just a quick idea - will try to test this later.)

**Comment by andreasjordan** [comment]
Was fixed with #7034.
